__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -479,6 +563,27 @@ Navigation.parameters = {
     },
 };
 
+export const OpensInANewTab: StoryComponentType = () => (
+    <View>
+        <View style={{padding: Spacing.large_24}}>
+            <Link href="#" target="_blank">
+                This is a Primary link that opens in a new tab
+            </Link>
+            <Link href="#" kind="secondary" target="_blank">
+                This is a Secondary link that opens in a new tab
+            </Link>
+        </View>
+    </View>
+);
+
+OpensInANewTab.parameters = {
+    docs: {
+        storyDescription: `When \`target="_blank"\`, the external icon is
+        automatically added to the end of the link. This indicates that the link
+        will open in a new tab.`,
+    },
+};
+
```

Comment:
*Super nit*: Can you add this story right before the `Variants` story? I personally prefer for all the variants in variants to be listed before the variants story if we're showing them separately, but it really doesn't matter either way.